src/app/Component/ChatScreen/chatscreen.js

```diff
@@ -46,7 +45,12 @@ class ChatScreen extends Component {
                         let details = [];
                         res.data.data.map((user) => {
                             if (user.username !== this.props.user.username) {
-                                details.push(user);
+                                {
```

Comment:
U can simply add users.messages && users.messages.length condition in render

Reply:
okay harish